optaplanner-quarkus-integration/optaplanner-quarkus/runtime/src/main/java/org/optaplanner/quarkus/config/SolverRuntimeConfig.java

```diff
@@ -33,4 +35,13 @@
      */
     @ConfigItem
     public TerminationRuntimeConfig termination;
+
+    /**
+     * Enable multithreaded solving for a single problem, which increases CPU consumption.
+     * Defaults to {@value SolverConfig#MOVE_THREAD_COUNT_NONE}.
+     * Other options include {@value SolverConfig#MOVE_THREAD_COUNT_AUTO}, a number
+     * or formula based on the available processor count.
+     */
+    @ConfigItem
+    public Optional<String> moveThreadCount;
```

Comment:
moveThreadCount should be before termination because in SolverConfig.java it's also in that order

Reply:
@Christopher-Chianelli This needs addressing before merge.